Documentation/Debugger/How To Debug MIEngine.md

```diff
@@ -73,17 +76,51 @@ MonoDevelop.exe
 5. Right-click the project node (blue square) and do "Options"
 6. Under Run -> Custom Commands, select "Execute" in the lower dropdown and choose a command (I use `c:\windows\notepad.exe` - it doesn't matter what the command is, but MonoDevelop requires it to exist before it'll light up the Run menu).
 
-#### Configure the extension to enable remote debugging
+##### Configure the extension to enable remote debugging
 
 Open the **~/.vscode/extensions/ms-vscode.cpptools-\<version\>/debugAdapters/OpenDebugAD7** file with a text editor and locate and uncomment the line at the bottom. When you start debugging, it will now hang until the remote debugger is attached from Xamarin Studio. 
 
-#### Attach the remote debugger
+##### Attach the remote debugger
 
 In MonoDevelop: Run -> Run With -> Custom Command Mono Soft Debugger
 Fill in the IP and port of the Linux/Mac OS X machine and hit "Connect" to start debugging
 
 After you've done this once, you can hit the MonoDevelop "Play" button or <kbd>F5</kbd> to bring up the connect dialog again.
 
+#### With VS Code + Mono Debug
+
+##### Install Prerequisites
+1. Install [VS Code](https://code.visualstudio.com/Download)
+2. Install Mono Debug extension for VS Code

```

Comment:
Why does this file not use "." consistently? Can you clean that up? It seems odd.